src/bin/languageserver/mod.rs

```diff
@@ -1208,9 +1153,14 @@ impl LanguageServer for SolangServer {
                     let range = loc_to_range(&loc, &hovers.file);
 
                     return Ok(Some(Hover {
-                        contents: HoverContents::Scalar(MarkedString::String(
-                            hover.val.to_string(),
-                        )),
+                        // contents: HoverContents::Scalar(MarkedString::String(
```

Comment:
Commented out code is almost never kept in the source (if there is a good exception, you can explain in a comment why it should be kept instead).